lib/features/persons/pages/persons_page.dart

```diff
@@ -36,64 +36,73 @@ class PersonsPage extends StatelessWidget {
               bloc.add(PersonsLoadEvent());
             }
             if (state is PersonsLoad) {
-              return Padding(
-                padding: const EdgeInsets.symmetric(
-                  vertical: Dimens.paddingXLarge,
-                  horizontal: Dimens.paddingLarge,
-                ),
-                child: Column(
-                  crossAxisAlignment: CrossAxisAlignment.start,
-                  children: [
-                    AppText(
-                      translate('task_add_people_subtitle'),
-                      type: TextTypes.subtitle,
-                    ),
-                    const SizedBox(height: Dimens.paddingMedium),
-                    CardButton(
-                      leading: const Icon(Icons.add),
-                      title: translate('Add person'),
-                      onTap: () {
-                        navigatorBloc.add(
-                          PushScreenNavigationEvent(
-                            model: NavigationModel(
-                              route: Routes.form,
-                              arguments: ArgsFormBuilderPage(formId: FormIds.personFormId),
-                            ),
-                          ),
-                        );
-                      },
-                    ),
-                    const SizedBox(height: Dimens.paddingXLarge),
-                    AppText(
-                      translate('persons_added'),
-                      type: TextTypes.titleMedium,
-                    ),
-                    const SizedBox(height: Dimens.paddingLarge),
-                    ListView.builder(
-                        shrinkWrap: true,
-                        primary: false,
-                        itemCount: state.persons.length,
-                        itemBuilder: (context, index) {
-                          Person person = state.persons[index];
-                          return Padding(
-                            padding: EdgeInsets.only(bottom: person == state.persons.last ? 0 : Dimens.paddingLarge),
-                            child: CardButton(
-                              title: (person.name + ' ' + person.surnames),
-                              trailing: const Icon(Icons.edit, color: PaletteColors.icons),
-                              onTap: () {
-                                navigatorBloc.add(
-                                  PushScreenNavigationEvent(
-                                    model: NavigationModel(
-                                      route: Routes.form,
-                                      arguments: ArgsFormBuilderPage(formId: FormIds.personFormId, personId: person.id),
-                                    ),
-                                  ),
-                                );
-                              },
+              return SingleChildScrollView(
```

Comment:
The only change here is that I wrapped everything with `SingleChildScrollView` because when there's a long list of people, it wouldn't be scrollable